parser.y

```diff
@@ -793,6 +793,7 @@ import (
 	IgnoreOptional			"IGNORE or empty"
 	IndexColName			"Index column name"
 	IndexColNameList		"List of index column name"
+	IndexColNameListOpt	    "List of index column name opt"
```

Comment:
```suggestion
	IndexColNameListOpt		"List of index column name opt"
```

Reply:
Indentation is fixed. Now these lines look like their neighborhood.